server/task/task-api/src/test/java/org/apache/james/task/TaskManagerContract.java

```diff
@@ -272,7 +272,6 @@ default void additionalInformationShouldBeUpdatedWhenCancelled(CountDownLatch co
 
         assertThat(additionalInformation.getCount())
             .isEqualTo(1);
-        countDownLatch.countDown();
```

Comment:
That commit should be a fixup

Reply:
It seems this piece of code is outdated though... Might be github mixing up the commits order again. Well maybe don't take it too seriously then !

Reply:
Yes when rebasing, github mess with the commits order. It's sad :(